### ISSUE TITLE
media indicator: offscreen text isn't translated

### ISSUE THREAD

**Issue by andrew-nowak** [bbc/simorgh#2096]
**Describe the bug**
![image](https://user-images.githubusercontent.com/10963046/60085283-eb18b480-9730-11e9-8168-1cbe3427d51c.png)

**To Reproduce**

1. Open a front page
2. Disable CSS
3. Scroll to a media promo

**Expected behavior**
'Audio' (or 'Video' in the above example) should be localised.

**Solutions**
1. Insert translations from service config
2. Don't place offscreen text in the media indicator. The media indicator only has offscreen text to support user agents which do not handle `aria-hidden: true`. The media indicator is also the _only part of simorgh which attempts to support those user agents._ We could just drop this functionality and rely on the offscreen text in the media promo.
